composer.json

```diff
@@ -16,11 +16,11 @@
 
     "require": {
         "php":                   ">=5.3.1",
-        "symfony/css-selector":  "~2.1|~3.0"
+        "symfony/css-selector":  "^2.7|^3.0|^4.0"
     },
 
     "require-dev": {
-        "symfony/phpunit-bridge": "~2.7|~3.0"
+        "symfony/phpunit-bridge": "^3.3|^4.0"
```

Comment:
You've increased `2.1` to `2.7` for `symfony/css-selector`. Why not doing something similar for 
 `symfony/phpunit-bridge` as well?

Reply:
The feature allowing to run both PHPUnit major versions has been added in 3.3, it doesn't work with earlier versions.

Reply:
I guess Symfony 3.x works fine on PHP 5.3 it Travis CI is happy.

Reply:
the PHPUnit bridge is special. We kept PHP 5.3 support in it (allowing Symfony 2.7 to use the latest bridge version for its own testing, instead of having to backport nice changes all the time).